pkg/schema/icon.go

```diff
@@ -13,21 +13,14 @@
 
 package schema
 
-import "github.com/kubeark/jsonschema"
+import (
+	"github.com/kubeark/jsonschema"
+)
 
 type icon struct{}
 
 func (icon) JSONSchema() *jsonschema.Schema {
 	return &jsonschema.Schema{
 		Type: &jsonschema.Type{Types: []string{"string"}},
-		AnyOf: []*jsonschema.Schema{
```

Comment:
Why removing this enum? Doesn't it apply anymore?

Reply:
The enum should only apply to `external.*.icon`, not the root-level `icon` which is only a string (URL)